config/formats.js

```diff
@@ -670,7 +670,7 @@ let Formats = [
 		maxLevel: 5,
 		ruleset: ['Obtainable', 'Standard' , 'Team Preview', 'Little Cup'],
 		banlist: ['Sneasel', 'Swirlix', 'Baton Pass'],
-                 requireGalar: true, 
+				requireGalar: true, 
```

Comment:
get rid of that tab so requireGalar will have the same space as banlist, ruleset, ...